src/authorization/WebAclAuthorizer.ts

```diff
@@ -121,7 +121,9 @@ export class WebAclAuthorizer extends Authorizer {
       this.logger.debug(`Trying to read the ACL document ${acl.path}`);
       const data = await this.resourceStore.getRepresentation(acl, { type: [{ value: INTERNAL_QUADS, weight: 1 }]});
       this.logger.info(`Reading ACL statements from ${acl.path}`);
-      return this.filterData(data, recurse ? ACL.default : ACL.accessTo, id.path);
+
+      const resourceId = await this.aclManager.isAcl(id) ? await this.aclManager.getResource(id) : id;
```

Comment:
Seems like we can just
```suggestion
      const resourceId = await this.aclManager.getResource(id);
```
with the current implementation (if that is correct)

Reply:
Are we sure this does not give Read access to an ACL file if we have Read access to its folder?

As in: this logic is restricted to only Control access?
Are we testing this?

Reply:
True, I think I changed that later in the manager and then didn't think of changing this then.

The only thing this part of the logic does is extracting all acl triples from an acl file related to this given identifier. The permission checking happens at https://github.com/solid/community-server/blob/9c78c4f9adbb83add6991723db52a15af0707108/src/authorization/WebAclAuthorizer.ts#L42-L44
where a different branch is taken if the input identifier is an acl file.